src/main/java/com/octopusdeploy/api/ReleasesApi.java

```diff
@@ -125,4 +127,28 @@ public String getPortalUrlForRelease(String projectId, String releaseVersion) th
         JSONObject links = json.getJSONObject("Links");
         return links.getString("Web");
     }
+
+    /**
+     * Get the partial Octopus portal URL for the latest release of a project
+     * @param projectId the id of the project to get the releases for
+     * @return A version of releases for a given project
+     * @throws IllegalArgumentException when the web client receives a bad parameter
+     * @throws IOException When the AuthenticatedWebClient receives and error response code
+     */
+    public String getPortalUrlForLatestRelease(String projectId) throws IllegalArgumentException, IOException {
+        AuthenticatedWebClient.WebResponse response = webClient.get("projects/" + projectId + "/releases");
+        if (response.isErrorCode()) {
+            throw new IOException(String.format("Code %s - %n%s", response.getCode(), response.getContent()));
+        }
+        JSONObject json = (JSONObject)JSONSerializer.toJSON(response.getContent());
+        JSONArray items = json.getJSONArray("Items");
```

Comment:
This looks accurate to me in terms of null checking 👍 If we don't have any error code, then `Items` should exist in the response. If we then have some items, we dig down through the JSON links to get the Web link.